module/vision/GoalDetector/src/GoalDetector.cpp

```diff
@@ -115,12 +120,18 @@ namespace vision {
                 MEASUREMENT_LIMITS_RIGHT = config["measurement_limits"]["right"].as<uint>();
                 MEASUREMENT_LIMITS_TOP   = config["measurement_limits"]["top"].as<uint>();
                 MEASUREMENT_LIMITS_BASE  = config["measurement_limits"]["base"].as<uint>();
+
+                vector3_covariance = config["vector3_covariance"].as<arma::vec>();
```

Comment:
Match the style of the other configuration parameters, either change all the others or these two :P